sigma/plugins/fun/xkcd.py

```diff
@@ -5,7 +5,7 @@
 
 async def xkcd(cmd, message, args):
 
-    comic_no = str(random.randint(1, 1724))
+    comic_no = str(random.randint(1, 2171))
```

Comment:
I can think of two ways:

1. Have a loop that checks to see if `http://xkcd.com/comic_no/` 404s. This is very inefficient: I tried it, and it takes about 30 seconds to run through the numbers up to 2000+

2. Add 1 to the max comic number every Monday, Wednesday, and Friday (because that's when the comic updates). This would leave us at the mercy of the author, though; if he misses a day, we will start getting 404s from normal requests. Maybe we can leave the number 10 or so behind, so that it has a greater amound of flexibility.

Reply:
Can't go to the latest comic, go to previous, get link, and add one to the comic id?

Reply:
Maybe we can, but I don't know how to make Python go to xkcd.com, click on [this button](https://puu.sh/DPuSg/c0c1f1b810.jpg), and get the link. I don't even know if it's possible.

Reply:
see https://realpython.com/python-requests/

Reply:
I don't think that's necessary, actually. I just found out that appending `/info.0.json` to `https://xkcd.com` gives the json info of the latest comic, and therefore the comic number. That makes things much easier.

Now, I just need to find a way to implement it...

Reply:
Fixed in 7c4fef6